### ISSUE TITLE
Reducible clause complements

### ISSUE THREAD

**Issue by nschneid** [nert-nlp/cgel#129]
p. 638:

<img width="539" alt="image" src="https://github.com/user-attachments/assets/9eb44068-5a81-488a-a26a-651b11a09451">

In our notation this would be:

```lisp
(PP
    :Head (P :t "although")
    :Comp (Clause
        :Head (VP
            :PredComp (AdjP [moderately rich]))))
```

Are we OK with a headless VP? Or would it be better to indicate the ellipsis somehow?

**Comment by BrettRey** [comment]
I think this confuses clausal semantics with clausal syntax. Just looks like a PP with an AdjP complement to me.

**Comment by nschneid** [comment]
Hmm...isn't there a great deal of overlap between what can be a canonical PredComp (e.g. with *become* or *seem*), and what can occur in this construction with *although* (and *if* and so on)?

- Although __careful/*carefully__, John missed a spot.
- Although **awake**, John was not paying attention.
- Although **behind schedule**, the project will not cost an inordinate amount.
- Although **a pain in the neck**, the problem can be solved.

Note that *despite* does NOT license these, nor does it license a full clause:

- *Despite __careful/he was careful__, John missed a spot.

Saying that *although* licenses a Clause, full or reduced—while *despite* does not—seems straightforward to me.

**Comment by BrettRey** [comment]
_for_ and _as_ license predicative complements. Why not _although_?

**Comment by nschneid** [comment]
In "see him as intelligent" or "take him for a fool", we would treat the entire PP as a PredComp, right? Whereas although-PPs are adjuncts.

**Comment by nschneid** [comment]
Oh I see, p. 636:
<img width="553" alt="image" src="https://github.com/user-attachments/assets/449213ca-d286-4ad1-9440-06973bae9998">

The full-clause paraphrase doesn't work, as pointed out in the first excerpt I posted. The question is whether this alternation with a full clause is important enough to be given common abstract structure in the tree, or whether it just reflects something historical.

**Comment by BrettRey** [comment]
I think it's best just to think of it as different possible complements.

**Comment by nschneid** [comment]
Are the same kinds of potentially clausal phenomena possible in as-PredComps and although-PredComps?

What about negation:
- As treasurer I recommend we increase the fees.
- *As not treasurer I recommend we increase the fees.
- Although treasurer, you do not have exclusive budgetary authority.
- Although not treasurer, you have exclusive budgetary authority.

Do you share these intuitions? If so they could be explained by saying there's a clause level in the "although" case that is not present with "as".

**Comment by BrettRey** [comment]
I do. That's a very nice observation. Anything else like that?

**Comment by nschneid** [comment]
I think adjuncts of reason go more readily with although-PPs:

- ?As treasurer because his father was, John is subject to accusations of nepotism.
- Although treasurer because his father was, John is highly qualified in his own right.

**Comment by nschneid** [comment]
What to do with:

- "What actions can be considered as betraying one's own country?"

Is this VP-as-PredComp? Clause-as-PredComp?

**Comment by BrettRey** [comment]
Yeah, I'm not sure. Same thing with predicative adjuncts like [_Owing money to my stupid bank,_] _I have to live very frugally_.
